src/PKSim.Core/Commands/SetRelativeExpressionCommand.cs

```diff
@@ -12,6 +12,13 @@ public SetRelativeExpressionCommand(IParameter parameter, double valueToSet)
          ObjectType = PKSimConstants.ObjectTypes.Molecule;
       }
 
+      protected override void UpdateParameter(IParameter parameter, IExecutionContext context)
+      {
+         base.UpdateParameter(parameter, context);
+         if (parameter == null) return;
+         parameter.IsDefault = (parameter.Value == 0);
```

Comment:
This is the big change here. We are doing a bunch of extra stuff for relative exprssions (normalizing etc..) . The easiest way to get a consistent behavio was to simply say that IsDefault means value is zero